recipes/graylog-forwarder/files/forwarder.conf

```diff
@@ -0,0 +1,5 @@
+forwarder_server_hostname =
+forwarder_grpc_tls_trust_chain_cert_file =
+forwarder_grpc_api_token =
+
+message_journal_dir = /var/lib/graylog-forwarder/journal
```

Comment:
We need to set some additional config fields:

```
bin_dir = /usr/share/graylog-forwarder/bin
plugin_dir = /usr/share/graylog-forwarder/plugin
data_dir = /var/lib/graylog-forwarder/data
node_id_file = /var/lib/graylog-forwarder/node-id
```

That way we don't rely on the defaults and the working directory.

Reply:
Fixed.